File: Mensuration/Reactangle and Cuboid/Perimeter/PoR.c
//Perimeter of a Rectangle
#include<stdio.h>
void main(){
    int l,b,peri;
    printf("Enter the Length : ");
    scanf("%d",&l);
    printf("Enter the Breadth : ");
    scanf("%d",&b);
    peri=2*(l+b);
    printf("Perimeter : %d",peri);
}

File: Mensuration/Reactangle and Cuboid/Perimeter/PoR.java
//Perimeter of a Rectangle
import java.util.Scanner;
public class PoR{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter Length : ");
        int l=sc.nextInt();
        System.out.print("Enter Breadth : ");
        int b=sc.nextInt();
        sc.close();
        int peri=2*(l+b);
        System.out.println("Perimeter : "+peri);
    }
}